### PR TITLE
(WIP) Migrate source maps

### PR DESCRIPTION
**Pull request by jasonLaster**
Issue: #5820

### Summary of Changes

* adds a new source-maps package to the debugger
* changes debugger dependency from devtools-source-map to source-maps

### Local Changes

* use yarn [workspaces][w] for local installs and symlink
* changes the debugger webpack config to point to the package

### Next Steps

- [ ] update `yarn test` and circleci
- [ ] document the release process
- [ ] backfill issues and prs

[w]: https://yarnpkg.com/lang/en/docs/workspaces/

### CODE REVIEW

**Comment by darkwing** [comment]
Where are we on this @jasonLaster and what's the priority?  Only `package.json` and `yarn.lock` have   been modified since, which are easy to update, but we don't want to let this go stale and inherit loads of merge conflicts.

**Comment by jasonLaster** [comment]
It's close. just have to test how the release process would go... but it's basically ready, next time i have some time

File: package.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "devtools-launchpad": "^0.0.119",
     "devtools-linters": "^0.0.4",
     "devtools-reps": "^0.22.0",
-    "devtools-source-map": "^0.16.0",
+    "devtools-source-map": "0.16.0",
```
Comment (nchevobbe) [inline package.json]: should we add a comment saying the version number will be _ignored_ because we use workspaces ?

Comment (nchevobbe) [reply package.json]: right 🤦‍♂️

**Pull request merged**